Makefile

```diff
@@ -26,7 +26,17 @@ deps-venv:
 
 compile-cairo: $(CAIRO_TARGETS)
 compile-starknet: $(STARKNET_TARGETS)
-	starknet-compile starknet_programs/account_without_validation.cairo  --account_contract > starknet_programs/account_without_validation.json
+	starknet-compile \
```

Comment:
Is there a way to make the compilation of the programs more generic?
Maybe we can move the `account_without_validation.cairo` to a different folder so we can compile it with the tag and leave the rest in starknet_programs

Reply:
Yes, fixed.